org.altaqwaa.Altaqwaa.yml

```
@@ -0,0 +1,41 @@
+app-id: org.altaqwaa.Altaqwaa
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+separate-locales: false
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: altaqwaa
+    buildsystem: simple
+    build-commands:
+      - mv * /app/bin
+      - mkdir -p /app/share/icons/hicolor/256x256/apps/
+      - mkdir -p /app/share/applications
+      - mkdir -p /app/share/metainfo
+      - mv /app/bin/${FLATPAK_ID}.png /app/share/icons/hicolor/256x256/apps/
+      - mv /app/bin/${FLATPAK_ID}.desktop /app/share/applications/
+      - install -Dm644 -t=/app/share/metainfo /app/bin/${FLATPAK_ID}.metainfo.xml
```

```suggestion
      - mv /app/bin/${FLATPAK_ID}.metainfo.xml /app/share/metainfo
```

since you are still set in installing that way.

---

Yeah, just noticed it as well. Thanks!